### PR TITLE
implemented key extraction feature using Gensim on #27

### PR DESCRIPTION
**Pull request by jkim3389**
#27

### CODE REVIEW

File: NLP test/testGensimNLP.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+from gensim.summarization.summarizer import summarize
```
Comment (adeeb897) [inline NLP test/testGensimNLP.py]: Should we keep the test file out of the repository?

File: crawler.py [hunk 2]
```diff
@@ -6,6 +6,7 @@
 import feedparser
 import time
 from id_crawling import study_id_crawling
+from gensim.summarization import keywords
```
Comment (adeeb897) [inline crawler.py:9]: Can you add this requirement to `requirements.txt` for us? You should just be able to add the line
`gensim==3.8.3`

**Review by adeeb897 — CHANGES_REQUESTED** [review]
Thanks for implementing it for us! I have some very minor suggestions. but otherwise it looks great.

File: crawler.py (hunk 2 above)
Comment (jkim3389) [reply crawler.py:9]: I will add it now

**Review by adeeb897 — APPROVED** [review]
Great

**Review by joheeju — APPROVED** [review]

**Pull request merged**